server/client_test.go

```diff
@@ -1997,9 +1998,13 @@ func TestNoClientLeakOnSlowConsumer(t *testing.T) {
 	}
 	defer c.Close()
 
-	var buf [1024]byte
+	var buf [4 * 1024]byte
 	// Wait for INFO...
-	c.Read(buf[:])
+	n, _ := c.Read(buf[:])
```

Comment:
I know that I wrote that test, but originally, the intent was just to make sure server sends something before client sends CONNECT.
Now, if you want to have inspect the INFO (to get the CID), you need to make sure we read it correctly. This call will does not. So we need to create a bufio.NewReader() and use ReadLine() instead?

Reply:
ok will take a look.

Reply:
ok pushed a fix. Please take a quick look.